rakelib/artifacts.rake

```diff
@@ -99,6 +99,8 @@ namespace "artifact" do
     @exclude_paths << 'vendor/**/gems/**/Gemfile'
 
     @exclude_paths << 'vendor/jruby/lib/ruby/gems/shared/gems/rake-*'
+    # exclude the maven jars until https://github.com/jruby/ruby-maven-libs/pull/4
+    @exclude_paths << 'vendor/bundle/jruby/**/gems/ruby-maven-libs-3.3.9'
```

Comment:
Why exclude `ruby-maven-libs-3.3.9` is it a typo and you meant `3.8.9` or this `3.3.9` are kicked in by something else?
In my local (in `vendor/bundle/jruby/3.1.0/gems/` ) I can see it contains:
```
ruby-maven-libs-3.3.9/  
ruby-maven-libs-3.8.9/
```

So I suspect  the 3.3.9 is undesired but not clear what picks it

Reply:
The goal is to not include the version that comes with JRuby. And from what I saw the files for the older version were still being left behind after the upgrade. This ensures the older version is not in the artifact.